packages/common/cache/interceptors/cache.interceptor.ts

```diff
@@ -64,15 +65,23 @@ export class CacheInterceptor implements NestInterceptor {
   }
 
   protected trackBy(context: ExecutionContext): string | undefined {
+    const cacheExcludeMetadata = this.reflector.get(
+      CACHE_EXCLUDE_METADATA,
+      context.getHandler(),
+    );
```

Comment:
Could we make this a `getAllAndMerge` to handle if `@CacheExclude()` is called on the class and not just the method? In case the interceptor is bound globally but a full controller should be skipped over.

```suggestion
    const cacheExcludeMetadata = this.reflector.getAllAndMerge(
      CACHE_EXCLUDE_METADATA,
      context.getClass(),
      context.getHandler(),
    );
```